src/pkg/scan/postprocessors/report_converters.go

```diff
@@ -290,3 +297,74 @@ func toVulnerabilityItem(record *scan.VulnerabilityRecord, artifactDigest string
 
 	return item
 }
+
+// updateReport updates the report summary with the vulnerability counts
+func (c *nativeToRelationalSchemaConverter) updateReport(ctx context.Context, vulnerabilities []*vuln.VulnerabilityItem, reportUUID string) error {
+	log.Debugf("Update report summary for report, uuid %v", reportUUID)
```

Comment:
log.G(ctx).WithFields(fields).Errorf("Update report summary for report, uuid %v", reportUUID)

Reply:
Updated